standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/cache/SharedCache.java

```diff
@@ -2476,6 +2499,25 @@ public boolean isTableConstraintValid(String catName, String dbName, String tblN
     return isValid;
   }
 
+  public boolean isTableCacheStale(String catName, String dbName, String tblName, String validWriteIdList) {
+    boolean isValid = false;
+
+    if (StringUtils.isEmpty(validWriteIdList))
+      return false;
```

Comment:
nit: add braces.

Also, should we return true? if validWriteId is not present, we can safely assume that

Reply:
Done